components/lib/media-query-helper.css

```
@@ -1,6 +1,7 @@
 @custom-media --viewport-small (width < 768px);
 @custom-media --viewport-medium (width < 1024px);
 @custom-media --viewport-large (width >= 1024px);
+@custom-media --viewport-x-large (width >= 1280px);
```

We need to consistently use either min-width breakpoints (>=) or max-width breakpoints (<=).

**Max-width breakpoints**

```suggestion
@custom-media --viewport-small (width < 768px);
@custom-media --viewport-medium (width < 1024px);
@custom-media --viewport-large (width < 1280px);
@custom-media --viewport-x-large (width < 1536px);
```

**Min-width breakpoints**

```suggestion
@custom-media --viewport-small (width > 768px);
@custom-media --viewport-medium (width > 1024px);
@custom-media --viewport-large (width > 1280px);
@custom-media --viewport-x-large (width > 1536px);
```

This will affect how we write responsive styles, [mobile-first or desktop-first](https://ishadeed.com/article/the-state-of-mobile-first-and-desktop-first/). [Tailwind is mobile-first](https://tailwindcss.com/docs/responsive-design#mobile-first) by default.

Just a note for now. We'll need to fix this across the board.

---

> We need to consistently use either min-width breakpoints (>=) or max-width breakpoints (<=).
> 
> **Max-width breakpoints**
> 
> **Min-width breakpoints**
> 
> This will affect how we write responsive styles, [mobile-first or desktop-first](https://ishadeed.com/article/the-state-of-mobile-first-and-desktop-first/). [Tailwind is mobile-first](https://tailwindcss.com/docs/responsive-design#mobile-first) by default.
> 
> Just a note for now. We'll need to fix this across the board.

Added a fix for breakpoints over here https://github.com/arcana-network/website/pull/128/commits/dc2c66454cbb022e5422e0c9270ee6ce9d4ce98c

Currently used `width < size` syntax (desktop first) because making it mobile first would require a lot of rework for now.